### PR TITLE
Create component C002

### PR DESCRIPTION
**Pull request by novadeveloper14**
In this task, component C002 was created and the layout of the first three cards of the creditProfileInfo page has been made.

### CODE REVIEW

**Review by LinconOssa — APPROVED** [review]
Meets the requirements.

File: src/components/cards/CardInfoContainer/CardInfoContainer.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      description: "title of the component",
+    },
+    children: {
+      description: "children of the component",
+    },
+  },
+};
+
+const Example = () => (
+  <Stack direction="column" gap="16px">
```
Comment (carin91) [inline src/components/cards/CardInfoContainer/CardInfoContainer.stories.tsx:26]: pass the gap value based on inube.spacing.s400 or its equivalent

File: src/components/cards/CardInfoContainer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  heigthCardInfoContainer?: string;
+}
+
+export const CardInfoContainer = (props: CardInfoContainerProps) => {
+  const {
+    title,
+    children,
+    aspectRatio,
+    icon,
+    heigthCardInfoContainer = "auto",
```
Comment (carin91) [inline src/components/cards/CardInfoContainer/index.tsx:19]: adjust the spelling: height

File: src/components/cards/CardInfoContainer/index.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  } = props;
+
+  return (
+    <Stack
+      direction="column"
+      width="-webkit-fill-available"
+      height={heigthCardInfoContainer}
+    >
+      <Stack gap={inube.spacing.s100} alignItems="center">
+        <Icon icon={icon} appearance="primary" size="56px" cursorHover />
```
Comment (carin91) [inline src/components/cards/CardInfoContainer/index.tsx:29]: Avoid cursor: pointer.

File: src/pages/board/outlets/creditProfileInfo/JobStabilityCard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  const { companySeniority, stabilityIndex, estimatedCompensation } = props;
+  return (
+    <CardInfoContainer
+      title="Estabilidad Laboral"
+      icon={<MdOutlineBusinessCenter />}
+      heigthCardInfoContainer="272px"
+    >
+      <Stack direction="column" gap={inube.spacing.s200}>
+        <Stack alignItems="center" gap={inube.spacing.s400}>
+          <Stack width="200px">
```
Comment (carin91) [inline src/pages/board/outlets/creditProfileInfo/JobStabilityCard/index.tsx:24]: Could you avoid those containers? i would try to work directly with the text, could you not?

File: src/pages/board/outlets/creditProfileInfo/JobStabilityCard/index.tsx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+            <Text size="medium">Antigüedad en la empresa</Text>
+          </Stack>
+          <Stack>
```
Comment (carin91) [inline src/pages/board/outlets/creditProfileInfo/JobStabilityCard/index.tsx:27]: Could you avoid those containers? i would try to work directly with the text, could you not?

**Review by carin91 — APPROVED** [review]

**Pull request merged**